source/administration/mmctl-cli-tool.rst

```diff
@@ -1057,76 +1073,173 @@ mmctl channel rename
    --local                       allows communicating with the server through a unix socket
    --strict                      will only run commands if the mmctl version matches the server one
 
-mmctl channel restore
-^^^^^^^^^^^^^^^^^^^^^
+mmctl channel search
+^^^^^^^^^^^^^^^^^^^^
 
 **Description**
 
-  Restore a previously deleted channel. Channels can be specified by ``[team]:[channel]`` (e.g., myteam:mychannel) or by channel ID.
+  Search a channel by channel name. Channel can be specified by team (e.g., ``--team myTeam myChannel``) or by team ID.
```

Comment:
Great catch, @justinegeffen! @mgdelacroix Can you please confirm case, please?

Reply:
Arg, sorry I completely missed this comment. It's just a case of the examples not being consistent, I can change it in `mmctl` directly and make it homogeneous as well

Reply:
No worries, I missed this comment and merged the PR. If the casing is correct as it stands and it's functionally accurate that's good. Since I merged this without confirming with you, LMK when that's done and I'll open a PR to fix this. :)